packages/truffle-contract/test/methods.js

```diff
@@ -1,28 +1,25 @@
 var assert = require("chai").assert;
 var BigNumber = require("bignumber.js");
-var util = require('./util');
+var util = require("./util");
```

Comment:
A bunch of the changes in this file came over from the stuff I cherry-picked from Chris' branch